src/docs/ftw-hosting/how-to-deploy-ftw-to-production/index.md

```diff
@@ -9,280 +9,39 @@ ingress:
 published: true
 ---
 
-## Getting started
```

Comment:
The old "Getting started" section can maybe stay as-is, except the Heroku-specific parts. Based on what I understood from Vesa, it's intended for a generic production deployment instruction anyway so we can maybe use it as a base